### PR TITLE
Repharsing secure connection strings as `aliases to connection string…

### PR DESCRIPTION
**Pull request by Scooletz**
Solves #144 

Requires updating doco.

### CODE REVIEW

**Comment by Scooletz** [comment]
@SeanFeldman Could you take a look is that what we meant?

File: src/Transport/Config/Extensions/AzureStorageAddressingSettings.cs [hunk 1]
```diff
@@ -7,30 +7,30 @@
 
     sealed class AzureStorageAddressingSettings
     {
-        internal void RegisterMapping(string defaultConnectionStringName, Dictionary<string, string> name2connectionString, bool shouldUseAccountNames)
+        internal void RegisterMapping(string defaultConnectionStringAlias, Dictionary<string, string> alias2connectionString, bool shouldUseAccountAliases)
```
Comment (SeanFeldman) [inline src/Transport/Config/Extensions/AzureStorageAddressingSettings.cs]: `alias2connectionString` --> `aliasToconnectionString`

Comment (SeanFeldman) [inline src/Transport/Config/Extensions/AzureStorageAddressingSettings.cs]: wait, that's a map, isn't it? So maybe `aliasToCconnectionStringMap`. Fancy :)

**Comment by SeanFeldman** [comment]
One minor comment. Exactly what we've talked about :)

**Comment by SeanFeldman** [comment]
@Particular/azure-storage-queues-maintainers if there are not objects, this can be merged

**Comment by Scooletz** [comment]
:+1:

**Comment by SeanFeldman** [comment]
Done

**Comment by Scooletz** [comment]
Thank you @SeanFeldman I'll be pushing another beta then. As soon as I have it, I'll move to https://github.com/Particular/docs.particular.net/pull/1797

**Comment by SeanFeldman** [comment]
Fantastic. I plan the same for ASB.

**Pull request merged**